intermine_boot/.travis.yml

```diff
@@ -0,0 +1,13 @@
+dist: xenial   #Use light Xenial distro
```

Comment:
Try adding support for running test on MacOS and Windows

Reply:
Okay sir!